docs/_includes/section-title.html

```diff
@@ -46,42 +45,33 @@ <h2 class="section-title__title">Today’s Deals – All With Free Shipping</h2>
 </div>
     {% endhighlight %}
 
-    <h3 id="section-title-cta">CTA Section Title</h3>
-    <p>The arrow icon is designed to act as a link to the primary group page.</p>
+    <h3 id="section-title-link">Linked Section Title</h3>
+    <p>An arrow icon gives visual affordance that the title is a link. A linked section title can also have a subtitle.</p>
 
     <div class="demo">
         <div class="demo__inner">
             <div class="section-title">
                 <div class="section-title__title-container">
                     <h2 class="section-title__title">
                         <a href="https://www.ebay.com">Today’s Deals – All With Free Shipping</a>
-                    </h2>
-                </div>
-                <div class="section-title__cta">
-                    <a href="https://www.ebay.com" tabindex="-1" aria-hidden="true">
-                        <span class="section-title__cta-text">See All</span>
-                        <svg class="icon icon--cta" focusable="false" height="24" width="24" aria-hidden="true">
-                            {% include symbol.html name="arrow-right-bold" %}
+                        <svg class="icon icon--arrow-right-extra-small" focusable="false" height="10" width="10" aria-hidden="true">
```

Comment:
This one seems strange how the link is on the text, but clicking on the svg does not trigger the link. I think more often than not, most people will try to click the svg button to navigate.

Reply:
We could move the SVG inside of the anchor, but then I always feel it looks a little weird when the underline shows under the icon on hover. I kind of feel the hit target and affordance of the heading is sufficient and a very quick learned behaviour. Let's check in with design offline.